en/diagnostics/STRDIAG010.md

```diff
@@ -0,0 +1,53 @@
+# Diagnostics Warning STRDIAG010
+
+> The Type '{0}' doesn't have a public parameterless constructor, which is needed for Serialization
+
+## Explanation
+
+By default, Stride serializers only support types that have a parameterless constructor.
```

Comment:
Make sure that each sentence (4 sentences in this case) follows the next one without a new line break. Ignore if I am wrong 🤣.

Reply:
did that